README.md

```diff
@@ -14,6 +14,16 @@ cd app
 yarn start
 ```
 
+## Glossary of Terms
+
+Document Type: Each file can be associated with a document type, or category, such as "Driver's License" or "Passport".
+
+File: A file is a digital representation of a document, such as jpgs, pngs, pdfs, etc. In IDEM, users can upload jpg and pdf files.
+
+Document: A document is a file that a user has uploaded and labeled with a document type. Documents can be used as evidence for various claim.
+
+Claim: A claim is a statement that the user is asserting as true. A claim can be verified at true by attaching evidence to it and completing the relevant verification process.
```

Comment:
A claim can be verified at true by attaching evidence

Did you mean to use the word "at" in the sentence above? or should it be "as"